deploy/resources/v0.8.0/02-clustertasks/buildah-task.yaml

```diff
@@ -0,0 +1,55 @@
+# auto generated by script/update-tasks.sh
+# DO NOT EDIT: use the script instead
+# source: https://raw.githubusercontent.com/openshift/tektoncd-catalog/release-v0.7/buildah/buildah.yaml
+#
+---
+apiVersion: tekton.dev/v1alpha1
+kind: ClusterTask
+metadata:
+  name: buildah
```

Comment:
are we planning to add version in names using the operator (buildah-v0.7.0, buildah-v0.8.0 ...)
or is it something we are going to handle in the future versions